orchestration/k8-airflow/k8-worker-requirements.in

```diff
@@ -0,0 +1,13 @@
+--constraint https://raw.githubusercontent.com/apache/airflow/constraints-2.9.1/constraints-3.10.txt
+apache-airflow[gcp]==2.9.1
+apache-airflow-providers-celery==3.6.2
+sshtunnel==0.4.0
+apache-airflow-providers-ssh==3.10.1
+apache-airflow-providers-mysql==5.5.4
+apache-airflow-providers-postgres==5.10.2
+google-cloud-secret-manager==2.20.0
+# extras
+psycopg2-binary==2.9.9
+dbt-core==1.8.6
+dbt-bigquery==1.8.1
+elementary-data[bigquery]==0.4.1
```

Comment:
on ne peut pas utiliser les même requirements que orchestration ? histoire de ne pas avoir trop de fichiers de requirements...

Reply:
non, il faut du commun a dbt et un executor celery

Reply:
on peut voir pour avoir 1 seul requirement orchestration avec qui serait le meme que dbt et a kube mais plus lourd

Reply:
il faut des packages en plus et beaucoup de package en moins et en regle générale on veut que les container soient les plus lightweight possible non ?

Reply:
ouais mais là on commence aussi à avoir des 3 fichiers de requirements qui ont une base commune, et ça va être relou à maintenir. 

Je pense pas que ce soit nécessaire de faire ça maintenant mais on pourrait probablement faire un 
`base_requirements.in` (qui contient nos pasckages communs) puis un 
* `k8-requirements.in` dont la première ligne est -r `base_requirements.in` et les packages spécifiques ensuite
* `orchestration-requirements.in` dont la première ligne est -r `base_requirements.in` et les packags au airflow local ensuite
* `dbt-requirements.in` dont la première ligne est -r `base_requirements.in` puis ceux  de dbt...

Y a ptet d'autre approches également, mais je pense qu'il faut faire attention à cette Dette Technique qui peut rapidement être relou (je mets un package à jour qqpart et j'oublie de l'update ailleurs)